umple/Umple-for-Vscode
# Imporve error messages with printed output and the links to the doc

[umple/umple#1986](https://github.com/umple/umple/issues/1986)
1.When generate files in other language from umple, a directory is created for better file management.
For example, java files are generated in a directory named “Java”. Related issue: [#1978](https://github.com/umple/umple/issues/1978) 
2.When compiling ump files or generated files, error and warning messages from java compiler would be printed in output channel. 
3.The message is changed back to "Success! Processed path\file.ump"  [#1869](https://github.com/umple/umple/issues/1869) 
4.The error code would become https URL in problem panel, it links to the manal page of umple for the errors or warnings

To test:
You may download a umple.jar in Umple-for-Vscode directory.
In VS code: 
`
code ./Umple-for-Vscode
`
Press F5 to start the extension development host
Compile or generate for a .ump file containing errors or warning. 
Check the output, problem panels for messages.
